src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -92,13 +93,22 @@ class FormPage extends React.Component {
       : this.props.form.data;
   };
 
-  goBack = () => {
+  goBack = (customPath = '') => {
```

Comment:
I was just thinking about this... should this function be called `goBack` anymore if it allows users to go other places as well?

Reply:
We could rename it since it could be confused with the `router` method. And it could take you to any page within the form - and this would be useful in another method we've been discussing about skipping chapters